packages/devextreme/js/__internal/ui/overlay/m_overlay.ts

```diff
@@ -1158,6 +1163,7 @@ const Overlay: typeof OverlayInstance = Widget.inherit({
     switch (name) {
       case 'animation':
         break;
+      case '_loopFocus':
```

Comment:
This property runtime change is supposed to be handled more explicit specific way. 
But I agree to leave it as is until this property is private and narrowly used